GUI_func

```diff
@@ -0,0 +1,65 @@
+from tkinter import *
+import time
+import morse_talk as mtalk
+import sys
+
+dc = ''
+cd = ''
+
+
+def gui():
+    def encode():
+        global dc
```

Comment:
Please visit this link [http://effbot.org/pyfaq/how-do-you-set-a-global-variable-in-a-function.htm](url).
Another point is that I have declared and initialised dc,cd twice so I am removing the local declaration of the variables in the function.